### ISSUE TITLE
Browsing history does not delete when container closed

### ISSUE THREAD

**Issue by ghost** [mozilla/multi-account-containers#2426]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

Use the product

### Actual behavior

Doesn't clear

### Expected behavior

Clear

### Additional informations

This isn't a bug, its a suggestions. Clicking the make a suggestion button gave me a life story to which I don't have time to read.

So basically, I have multiple e-mail accounts with a single e-mail service provider. I login to one e-mail account in a container, close the container, go to login to the second e-mail account, and find the site data like cookies was not cleared when the container was closed.

I reckon there should be an option to change this.

**Comment by dannycolin** [comment]
This is the expected behavior. Each container has it's own cookie jar that is permanent until you purge all cookies in the Firefox settings.

Also closing as a duplicate of #48. 

Please next time, take the time to search for other similar bugs or use connect.mozilla.org to propose new feature. I do understand that you may not have unlimited free time but there's volunteers like me working on this project for free that don't have a lot of spare time either to triage the bugs that aren't bug.
